deployment/amplify/production/amplify/.config/project-config.json

```diff
@@ -0,0 +1,20 @@
+{
+    "projectName": "src",
+    "projectPath": "/usr/local/src/production/",
+    "defaultEditor": "sublime",
+    "frontendHandler": {
+        "javascript": "/usr/local/lib/node_modules/@aws-amplify/cli/node_modules/amplify-frontend-javascript"
+    },
+    "javascript": {
+        "framework": "react",
+        "config": {
+            "SourceDir": ".",
```

Comment:
This is a somewhat hacky way to get the `aws-exports.js` file (which the frontend code needs in order to ping AWS resources) to write to the `deployment/amplify/${env}` directory. The Amplify CLI will follow this path to write `aws-exports.js`, whether or not it's the true source directory.